### ISSUE TITLE
Elastic Search 6.x Post install step would be stuck on upgrade

### ISSUE THREAD

**Issue by PikachuEXE** [Homebrew/homebrew-core#32727]
- [x] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
brew upgrade elasticsearch
- What happened (include command output)
```
→ brew upgrade elasticsearch
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
No changes to formulae.

==> Upgrading 1 outdated package, with result:
elasticsearch 6.4.2
==> Upgrading elasticsearch
==> Downloading https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-6.4.2.tar.gz
Already downloaded: /users/pikachuexe/Library/Caches/Homebrew/downloads/819c3bef14b385b1b8128e414bd1138285bad364e365231075e77fc92646a7a9--elasticsearch-oss-6.4.2.tar.gz
==> /usr/local/Cellar/elasticsearch/6.4.2/bin/elasticsearch-keystore create
(stuck here)
```
`brew postinstall elasticsearch` also won't work
Running `/usr/local/Cellar/elasticsearch/6.4.2/bin/elasticsearch-keystore create` would show a prompt:
```
$ /usr/local/Cellar/elasticsearch/6.4.2/bin/elasticsearch-keystore create
An elasticsearch keystore already exists. Overwrite? [y/N]
```
- What you expected to happen
The post install should not be stuck and succeed on upgrade
- Step-by-step reproduction instructions (by running `brew install` commands)
1. Install elasticsearch through brew (I think that will create the keystore for your)
2. Uninstall it
3. Install it again

`brew gist-logs`:
```
$ brew gist-logs elasticsearch
https://gist.github.com/d0972c37b32e91b7b870f574e4300cdd
```

**Comment by chrmoritz** [comment]
Thanks for the report. This should be fixed in #32750.

**Comment by fxcoudert** [comment]
Thanks @PikachuEXE for the bug report!
